src/ru/nsu/ccfit/zuev/osu/scoring/ScoringScene.java

```diff
@@ -490,7 +499,9 @@ public boolean onAreaTouched(final TouchEvent pSceneTouchEvent,
                 || stat.getMod().contains(GameMod.MOD_SCOREV2)
                 || Config.isRemoveSliderLock()
                 || ModMenu.getInstance().isChangeSpeed()
-                || ModMenu.getInstance().isEnableForceAR()){
+                || ModMenu.getInstance().isEnableForceAR()
+                || Float.compare(ModMenu.getInstance().getFLfollowDelay(), FlashLightEntity.defaultMoveDelayS) != 0
```

Comment:
Ditto.

Reply:
Thinking, i did this because i was meaning to make just the 120ms option ranked (future) but yeah i can change this, should i?

Reply:
Yeah, I don't think it matters now since the mod itself is unranked, and that's what matters.

Reply:
k